Gemfile

```diff
@@ -3,3 +3,4 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in contribute.gemspec
 gemspec
+gem 'contribute'
```

Comment:
This line isn't needed. Why would you require the gem which you are making itself. This is also causing the build failure.